cicd/gitops/configuring-sso-on-argo-cd-using-dex.adoc

```diff
@@ -8,6 +8,8 @@ toc::[]
 
 After the {gitops-title} Operator is installed, Argo CD automatically creates a user with `admin` permissions. To manage multiple users, cluster administrators can use Argo CD to configure Single Sign-On (SSO).
 
+IMPORTANT: The `spec.dex` parameter in the ArgoCD CR is deprecated. In the future release of Red Hat Openshift GitOps v1.9, configuring Dex using the `spec.dex` parameter in the ArgoCD CR is planned to be removed. Consider using the `.spec.sso` parameter instead.
```

Comment:
As you have added the note here, shall we remove the note from [Disabling Dex](https://57800--docspreview.netlify.app/openshift-enterprise/latest/cicd/gitops/configuring-sso-on-argo-cd-using-dex.html#gitops-disable-dex_configuring-sso-for-argo-cd-using-dex) section?

Reply:
@varshab1210 - That note is specific to the DISABLE_DEX parameter and its replacement `.spec.sso.dex` parameter. I would prefer retaining it as we will be removing it in v1.9 anyways.